tracee-ebpf/Makefile

```diff
@@ -145,10 +147,13 @@ mostlyclean:
 
 .PHONY: clean
 clean:
-	-$(CMD_DOCKER) rmi $(file < $(docker_builder_file))
+	-FILE="$(docker_builder_file)" ; \
+	if [ -n "$$FILE" ] && [ -r "$$FILE" ] ; then \
```

Comment:
I believe -n is redundant if -r is checked no?
(applies to other occurrence as well)

Reply:
It is defensive programming from a more elegant era that no longer exists. I will remove it.